dom-renderer.jsx

```diff
@@ -116,6 +116,8 @@ class DomItem extends THREE.Object3D {
   destroy() {
     physicsScene.enableActor(this.physicsObject);
     physicsScene.removeGeometry(this.physicsObject);
+    this.remove(this.floatNode);

```

Comment:
This line doesn't seem needed, since this Object3D is not valid for use in the scene after calling `destroy()` anyway.